docs/class5/connexml.md

```diff
@@ -484,6 +487,174 @@ An existing call is transferred to a different destination using the `Dial` ver
             </Dial>
         </Response>
         ```
+    15. **statusCallbackEvent, statusCallback, statusCallbackMethod**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Client
+                statusCallbackEvent="initiated ringing answered completed"
+                statusCallback="http://fr1js1.connexcs.net:3002"
```

Comment:
do not use fr1js1.connexcs.net add an example uri

Reply:
corrected: statusCallback="https://api.example.com/compile?rateCompact=true&ids=101,202,303"
                statusCallbackMethod="POST">